The test don't work. Can someone give me a hint how to get it working?

```
checking for references to /tmp/nix-build-python3.6-micawber-0.3.4.drv-0 in /nix/store/66p8373pfr4p2spr6y0kc39y88wvw8zf-python3.6-micawber-0.3.4...
running install tests
running test
running egg_info
writing micawber.egg-info/PKG-INFO
writing dependency_links to micawber.egg-info/dependency_links.txt
writing top-level names to micawber.egg-info/top_level.txt
reading manifest file 'micawber.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
writing manifest file 'micawber.egg-info/SOURCES.txt'
running build_ext
runtests (unittest.loader._FailedTest) ... ERROR

======================================================================
ERROR: runtests (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: runtests
Traceback (most recent call last):
  File "/nix/store/87wc186a9xq5y6yxvar6f2sm5cbqzpzr-python3-3.6.4/lib/python3.6/unittest/loader.py", line 153, in loadTestsFromName
    module = __import__(module_name)
ModuleNotFoundError: No module named 'runtests'


----------------------------------------------------------------------
Ran 1 test in 0.000s

FAILED (errors=1)
Test failed: <unittest.runner.TextTestResult run=1 errors=1 failures=0>
error: Test failed: <unittest.runner.TextTestResult run=1 errors=1 failures=0>
builder for ‘/nix/store/vigdmz21p511z4jfwzk6wa2r3avl1pc5-python3.6-micawber-0.3.4.drv’ failed with exit code 1
error: build of ‘/nix/store/vigdmz21p511z4jfwzk6wa2r3avl1pc5-python3.6-micawber-0.3.4.drv’ failed
```

It may just need one dependency, but i don't know.

https://github.com/coleifer/micawber/blob/master/setup.py#L40

--- a/pkgs/development/python-modules/micawber/default.nix
+++ b/pkgs/development/python-modules/micawber/default.nix
@@ -1,0 +1,26 @@
+{ stdenv, buildPythonPackage, fetchPypi, beautifulsoup4 }:
+
+buildPythonPackage rec {
+  pname = "micawber";
+  version = "0.3.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0pnq6j8f144virhri0drgf0058x6qcxfd5yrb0ynbwr8djh326yn";
+  };
+
+  propagatedBuildInputs = [ beautifulsoup4 ];
+
+  meta = with stdenv.lib; {
+    homepage = http://micawber.readthedocs.io/en/latest/;
+    description = "A small library for extracting rich content from urls";
+    license = licenses.mit;
+    longDescription = ''
+      micawber supplies a few methods for retrieving rich metadata
+      about a variety of links, such as links to youtube videos.
+      micawber also provides functions for parsing blocks of text and html
+      and replacing links to videos with rich embedded content.
+    '';
+    maintainers = with maintainers; [ davidak ];
+  };
+}

--- a/pkgs/top-level/python-packages.nix
+++ b/pkgs/top-level/python-packages.nix
@@ -10111,6 +10111,8 @@ in {
     };
   };
 
+  micawber = callPackage ../development/python-modules/micawber { };
+
   minimock = buildPythonPackage rec {
     version = "1.2.8";
     name = "minimock-${version}";

